examples/external_request/simple.rb

```diff
@@ -0,0 +1,75 @@
+# This example is one of the simplest examples demonstrating sending one span 
+# at at time to the Trace API endpoint.  It will time how long it takes to retrive
+# an external URL's content and construct a span and send it to New Relic's server.
+# 
+# Usage: `API_KEY=<YOUR_API_KEY bundle exec ruby simple.rb`
+require 'net/http'
+require 'bundler'
+Bundler.require
+
+require "new_relic/telemetry_sdk"
+
+# Fail fast if an API_KEY wasn't exported to the environment.
+unless ENV["API_KEY"]
+  raise "No API Key supplied.  Export API_KEY environment variable!" 
+end
+
+# Just a random ID generator -- A string of 16 (by default) hexidemal digits.
+def random_id length=16
+  length.times.map{rand(16).to_s(16)}.join
+end
+
+# Instantiates and memoizes the SDK client for sending Spans
+def span_client
+  return @span_client if @span_client
+  @span_client = NewRelic::TelemetrySdk::SpanClient.new
+end
+
+# Wraps the given block by recording time to retrieve URL's resource, then
+# constructs the span object and sends it to the server.
+# The response from the given block is returned as the method's result
+def record_external_request
+  # Capture time elapsed to make the external request (given block)
+  start_time = Time.now
+  response = yield
+  finish_time = Time.now
+  duration = finish_time - start_time
+
+  # Set some custom attributes for the Span (some static, some dynamic)
+  custom_attributes = {
+    "name": "Net::HTTP#get",
+    "http.method": "GET",
+    "http.status": response.code,
+    "size": response.body.bytesize,
+  }
+
+  # Construct the span and send it.
+  span = NewRelic::TelemetrySdk::Span.new(
+    id: random_id(8),
+    trace_id: random_id(16),
```

Comment:
Please change these to be 16 and 32 characters, respectively, to better reflect our actual defaults (now that that bug has been addressed).